internal/cache/file/downloader/job.go

```diff
@@ -37,12 +36,11 @@ import (
 type jobStatusName string
 
 const (
-	NOT_STARTED jobStatusName = "NOT_STARTED"
-	DOWNLOADING jobStatusName = "DOWNLOADING"
-	COMPLETED   jobStatusName = "COMPLETED"
-	FAILED      jobStatusName = "FAILED"
-	CANCELLED   jobStatusName = "CANCELLED"
-	INVALID     jobStatusName = "INVALID"
+	NotStarted  jobStatusName = "NotStarted"
```

Comment:
can we rename it to jobStatusName to jobState?

Reply:
We can skip for now, since we are in time-crunch.

Reply:
Also, I would prefer creating a new PR for that since this is already a long PR and focus is on cleaning up completed job.